proto/src/google/ground/v1beta1/survey.proto

```diff
@@ -37,6 +37,18 @@ message Survey {
 
   // Required. User id of the survey owner.
   string owner_id = 5;
+
+  // Properties related to sharing of survey data.
+  message DataSharing {
```

Comment:
Maybe `DataSharingConsent dataSharingConsent` and `DataSharingConsentType`? Nouns for messages are preferable and making the "consent" part explicit differentiates it from ACLs.

Reply:
Sounds good! Updated the proto names per that suggestion.